src/layouts/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function Layout(frontMatter) {
```

Comment:
this was throwing me off. templates are functions that return a component function and deliver frontmatter